src/frontend/apps/impress/src/features/docs/doc-header/components/DocVersionHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import { useTranslation } from 'react-i18next';
+
+import { Box, Text } from '@/components';
+import { HorizontalSeparator } from '@/components/separators/HorizontalSeparator';
+import { useCunninghamTheme } from '@/cunningham';
+import { Doc } from '@/features/docs/doc-management';
+
+interface DocHeaderProps {
+  doc: Doc;
+}
+
+export const DocVersionHeader = ({ doc }: DocHeaderProps) => {
+  const { spacingsTokens } = useCunninghamTheme();
+
+  const spacings = spacingsTokens();
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Box
+        $width="100%"
+        $padding={{ vertical: 'base' }}
+        $gap={spacings['base']}
+        aria-label={t('It is the document title')}
+      >
+        <Text as="h2" $margin={{ all: 'none', left: 'none' }} $size="h2">
+          {doc.title}
+        </Text>
```

Comment:
I don't know if it is a good things to have the title in 2 different places. 
Could this part be a small component coming `DocTitle.tsx`? if we do a refacto, we will have less chance to forget this one as well.

```suggestion
import {DocTitleVersion} from "./DocTitle.tsx";
        <DocTitleVersion title={title}>
```